src/HealthGPS/analysis_module.cpp

```diff
@@ -326,28 +326,64 @@ void AnalysisModule::calculate_population_statistics(RuntimeContext &context,
     // Calculate in-place averages
     for (auto i = min_age; i <= max_age; i++) {
         for (const auto &chan : series.channels()) {
-            if (chan == "count") {
+            if (chan == "count" || chan.starts_with("std_")) {
```

Comment:
Not really a suggestion, but why isn't this just
```suggestion
            if (!chan.starts_with("mean_")) {
```

Reply:
Ah, right, so some of the columns are binary variables, which are normalised to fractions using the division below. Like the disease `incidence_` and `prevalence_` fields. Maybe a better comment/name for this would be computing means *and* fractions.